### PR TITLE
Node JS updates for Trusty

### PR DESCRIPTION
**Pull request by plaindocs**
Move version info into reference

### CODE REVIEW

File: user/languages/javascript-with-nodejs.md [hunk 1]
```diff
@@ -4,9 +4,33 @@ layout: en
...
 ---
 
-<div id="toc"></div>
+<div id="toc">
+</div>
 
-This guide covers build environment and configuration topics specific to Node.js
+<aside markdown="block" class="ataglance">
+
+| JS and Node                                 | Default                                   |
```
Comment (BanzaiMan) [inline user/languages/javascript-with-nodejs.md]: This should be "JavaScript and Node.js", I think.

Comment (plaindocs) [inline user/languages/javascript-with-nodejs.md]: Yeah, we've not been too consistent before - I'll fix up the title as well.

File: user/languages/javascript-with-nodejs.md [hunk 2]
```diff
@@ -245,60 +293,18 @@ Find the source code at [travis-ci-meteor-packages](https://github.com/arunoda/t
...
-.travis.yml](https://github.com/scottyapp/hook.io-amqp-listener/blob/master/.travis.yml).
-
-For precise versions pre-installed on the VM, please consult "Build system
-information" in the build log.
-
 ## Node.js v4 (or io.js v3) compiler requirements
 
-To compile native modules for io.js v3 or Node.js v4 or later, a
-[C++11 standard](https://en.wikipedia.org/wiki/C%2B%2B11)-compliant compiler is required. More specifically, either gcc 4.8 (or later), or clang 3.5 (or later) works.
+To compile native modules for io.js v3 or Node.js v4 or older, a [C++11
```
Comment (BanzaiMan) [inline user/languages/javascript-with-nodejs.md]: Is the switch to "older" intentional?

Comment (plaindocs) [inline user/languages/javascript-with-nodejs.md]: It was, but perhaps I was mistaken. It followed on from the old Node versions topic. But this is actually compiling modules not older versions.

**Pull request merged**